pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix

```diff
@@ -0,0 +1,1374 @@
+    language = "teal";
+    version = "1ae8c68";
+    source = fetchFromGitHub {
+      owner = "euclidianAce";
+      repo = "tree-sitter-teal";
+      nativeBuildInputs = [ nodejs tree-sitter ];
+      postFetch = "pushd $out && tree-sitter generate && popd";
+      rev = "1ae8c68e90523b26b93af56feb7868fe4214e2b2";
+      hash = "sha256-8FqXrsgOI/cVezgVsQqPLlTa4Qnzmbuelo0XsI48fV0=";
+    };
+  };
+  tiger = buildGrammar {
+    language = "tiger";
+    version = "eb1d371";
+    source = fetchFromGitHub {
+      owner = "ambroisie";
+      repo = "tree-sitter-tiger";
+      rev = "eb1d3714998977ae76ca7c6a102b10ee37efc2b5";
+      hash = "sha256-kbwERaTHk5Pj5AfpbXPuRS6speB+xLMfrhRXTVOyMNw=";
+    };
+  };
+  tlaplus = buildGrammar {
+    language = "tlaplus";
+    version = "deaf0e5";
+    source = fetchFromGitHub {
+      owner = "tlaplus-community";
+      repo = "tree-sitter-tlaplus";
+      rev = "deaf0e5c573ad4e2bbfc9a29abb7b6dcb572556e";
+      hash = "sha256-D4A2k14SpVR4iKCMwql403XjHGg7p17EYazvAUiJ2gY=";
+    };
+  };
+  todotxt = buildGrammar {
+    language = "todotxt";
+    version = "0207f6a";
+    source = fetchFromGitHub {
+      owner = "arnarg";
+      repo = "tree-sitter-todotxt";
+      rev = "0207f6a4ab6aeafc4b091914d31d8235049a2578";
+      hash = "sha256-FWVW+qWOUSri+qf+qistf8bKWJCTxUKkoO66yUYtwHI=";
+    };
+  };
+  toml = buildGrammar {
+    language = "toml";
+    version = "8bd2056";
+    source = fetchFromGitHub {
+      owner = "ikatyang";
+      repo = "tree-sitter-toml";
+      rev = "8bd2056818b21860e3d756b5a58c4f6e05fb744e";
+      hash = "sha256-z9MWNOBxLHBd/pVs5/QiSSGtaW+DUd7y3wZXcl3hWnk=";
+    };
+  };
+  tsx = buildGrammar {
+    language = "tsx";
+    version = "0ab9d99";
+    source = fetchFromGitHub {
+      owner = "tree-sitter";
+      repo = "tree-sitter-typescript";
+      rev = "0ab9d99867435a7667c5548a6617a6bf73dbd830";
+      hash = "sha256-Nx+K7Ic/ePKAXPIMlrRn6zELYE59f/FnnZ/LM5ELaU8=";
+    };
+    location = "tsx";
+
+  };
+  turtle = buildGrammar {
+    language = "turtle";
+    version = "085437f";
+    source = fetchFromGitHub {
+      owner = "BonaBeavis";
+      repo = "tree-sitter-turtle";
+      rev = "085437f5cb117703b7f520dd92161140a684f092";
+      hash = "sha256-ub777Pjody2SvP2EjW7IwWj8YnMuMzdJ4AlrkP6WrdA=";
+    };
+  };
+  twig = buildGrammar {
+    language = "twig";
+    version = "6a01f20";
+    source = fetchFromGitHub {
+      owner = "gbprod";
+      repo = "tree-sitter-twig";
+      rev = "6a01f20e73038300d205d370212c361949be3035";
+      hash = "sha256-M+57mGl4Sgn0yUGAyxHxE6ShR+E/ki4m8/x/f/KHetA=";
+    };
+  };
+  typescript = buildGrammar {
+    language = "typescript";
+    version = "0ab9d99";
+    source = fetchFromGitHub {
+      owner = "tree-sitter";
+      repo = "tree-sitter-typescript";
+      rev = "0ab9d99867435a7667c5548a6617a6bf73dbd830";
+      hash = "sha256-Nx+K7Ic/ePKAXPIMlrRn6zELYE59f/FnnZ/LM5ELaU8=";
+    };
+    location = "typescript";
+
```

Comment:
The generator could be a little bit nicer and remove this new line.

Reply:
I will fix this in a new pr